pkg/controller/controller.go

```diff
@@ -435,11 +453,17 @@ func (ctrl *resizeController) resizeVolume(
 		return newSize, fsResizeRequired, fmt.Errorf("resize volume %s failed: %v", pv.Name, err)
 	}
 	klog.V(4).Infof("Resize volume succeeded for volume %q, start to update PV's capacity", pv.Name)
+	updatedPV, err := util.UpdatePVCapacity(pv, newSize, ctrl.kubeClient)
 
-	if err := util.UpdatePVCapacity(pv, newSize, ctrl.kubeClient); err != nil {
+	if err != nil {
 		klog.Errorf("Update capacity of PV %q to %s failed: %v", pv.Name, newSize.String(), err)
 		return newSize, fsResizeRequired, err
 	}
+	err = ctrl.volumes.Update(updatedPV)
+	if err != nil {
+		klog.Errorf("error updating PV %s: %v", updatedPV.Name, err)
```

Comment:
Can we return the error and let the caller log?

Reply:
done.